docs/source/index.rst

```diff
@@ -60,6 +60,6 @@ Currently it supports:
 * :doc:`Performance tuning [Experimental] <performance>`
 * :doc:`Upgrade procedures <upgrade>`
 * :doc:`Releases <releases>`
-* :doc:`Known issues <known-issues>`
+* :doc:`Known issues <support/known-issues>`
```

Comment:
Shouldn't this point to `Support`, instead of just this one subpage?

Reply:
Off topic to this PR, but I'm also thinking if we really need this "Choose a topic to begin" section, if all of these docs are listed in a sidebar anyway?

Reply:
sidebar is kind of unusable for that (see https://github.com/scylladb/sphinx-scylladb-theme/issues/929 ) but yeah, we need to restructure a bit in general

Reply:
> sidebar is kind of unusable for that (see https://github.com/scylladb/sphinx-scylladb-theme/issues/929 ) but yeah, we need to restructure a bit in general

But the "Choose a topic to begin" on the main page doesn't really help this either? I don't think we're on the same page here, my point is there isn't really anything added by listing everything there - but again, that's out of scope of this PR. 

> Shouldn't this point to Support, instead of just this one subpage?

(but this is in scope and still stands)

Reply:
updated